linux/step01_debian9_pg_deps.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PGVER=${PGVER:-pg96}
+
+# Postgres installation
+# 9.6
+#start-recommended
+apt-get -y install postgresql
```

Comment:
this is the version installed when running ``postgresql``
in other installations doc we have not been explicit unless we have to